Translator.php

```diff
@@ -35,9 +35,10 @@
  */
 class Translator extends BaseObject
 {
-    private $_where;
-    private $_params = [];
-    private $_operators;
+    protected $_where;
+    protected $_params = [];
+    protected $_operators;
+    protected $filteredByFields = [];
```

Comment:
why $filterByFields property?

Reply:
need to know what rules was applied

Reply:
Intercept request 'rules' param to check it

Reply:
rules  has nested  hierarchy (add group )

Reply:
yes, but a recursive method solves it

Reply:
buildWare already does recursion - just save additional info from it